Framework/API/src/MatrixWorkspace.cpp

```diff
@@ -83,6 +84,31 @@ MatrixWorkspace::~MatrixWorkspace() {
   }
 }
 
+void MatrixWorkspace::setIndexTranslator(
```

Comment:
I guess you are right. I had also considered having
```cpp
template <class... T>
void setIndexTranslator(T &&... args);
```
and then forward to the constructor. However, that would imply that we need to include `IndexTranslator.h` in `MatrixWorkspace.h`. What is your opinion on this?